tests/testthat/test-stroke.R

```diff
@@ -24,15 +24,14 @@ test_that("a stroke is found in a very simple network", {
   #           p4
   #         /
   # p1 - p2 - p3
-
   expected <- sf::st_sfc(sf::st_linestring(c(p1, p2, p3)), l3)
   actual <- stroke(sfc)
   expect_setequal(actual, expected)
 })
 
 test_that("a ring is recognized as a stroke", {
   sfc <- sf::st_sfc(l2, l4, l6, l7)
-  expected <- sf::st_sfc(sf::st_linestring(c(p3, p6, p5, p2, p3)))
+  expected <- sf::st_sfc(sf::st_linestring(c(p2, p3, p6, p5, p2)))
```

Comment:
Having slightly modified `merge_lines`, created a small difference on how strokes are constructed, so the ring now starts at a different position